test/extensions/common/aws/signer_impl_test.cc

```diff
@@ -211,6 +211,48 @@ TEST_F(SignerImplTest, SignHeadersNonS3) {
             headers.get(SignatureHeaders::get().ContentSha256)->value().getStringView());
 }
 
+// Verify signing headers for es
+TEST_F(SignerImplTest, SignHeadersES) {
```

Comment:
Do you think having a helper here will be helpful? For example: `SignerImplTest::expectSignHeaders(absl::string_view service_name, absl::string_view signature)` with the service name and the expected signature as params.

Reply:
That looks good!

Reply:
added expectSignHeaders. 😄